src/com/palmergames/bukkit/towny/command/NationCommand.java

```diff
@@ -1365,7 +1372,7 @@ public void mergeNation(Player player, String name) throws TownyException {
 			});
 			
 			// Send confirmation.
-			ConfirmationHandler.sendConfirmation(BukkitTools.getPlayerExact(king.getName()), confirmation);
+			ConfirmationHandler.sendConfirmation(player, confirmation);

```

Comment:
Your branch of confirmations overhaul was missing the commit I made this morning.

Reply:
Pretty *sure* it's fixed now

Reply:
Still not fixed.